File: test/fixtures/csharp/multiline_post_with_data.cs
using System.Net.Http;
using System.Net.Http.Headers;

HttpClient client = new HttpClient();

HttpRequestMessage request = new HttpRequestMessage(HttpMethod.Get, "http://localhost:28139/echo/html/");

request.Headers.Add("Origin", "http://fiddle.jshell.net");

request.Content = new StringContent("msg1=value1&msg2=value2");
request.Content.Headers.ContentType = new MediaTypeHeaderValue("application/x-www-form-urlencoded");

HttpResponseMessage response = await client.SendAsync(request);
response.EnsureSuccessStatusCode();
string responseBody = await response.Content.ReadAsStringAsync();

File: test/fixtures/clojure/multiline_post_with_data.clj
(require '[clj-http.client :as client])

(client/get "http://localhost:28139/echo/html/" {:headers {"Origin" "http://fiddle.jshell.net"}
                                                 :form-params {:msg1 "value1"
                                                               :msg2 "value2"}})
